aws-greengrass-testing-standalone/README.md

```diff
@@ -32,14 +32,18 @@ java \
 The results of the run can be found in `hello_world/test-results`.
 
 ## Run provided features
-You can also run features provided by this test framework.
-Here are the provided features: 
-- [cloudComponent.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-cloudcomponent/src/main/resources/greengrass/features/cloudComponent.feature)
-- [docker.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-docker/src/main/resources/greengrass/features/docker.feature)
-- [localdeployment.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-localdeployment/src/main/resources/greengrass/features/localdeployment.feature)
-- [ml.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-ml/src/main/resources/greengrass/features/ml.feature)
-- [mqtt.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-mqtt/src/main/resources/greengrass/features/mqtt.feature)
-- [streammanager.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-streammanager/src/main/resources/greengrass/features/streammanager.feature)
+You can also run features provided by this test framework. Here are the provided test features: 
+- Basic test features:
+  - [cloudComponent.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-cloudcomponent/src/main/resources/greengrass/features/cloudComponent.feature): Validates device capability for cloud components. 
+  - [localdeployment.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-localdeployment/src/main/resources/greengrass/features/localdeployment.feature): Validates device capability for local components.
+  - [mqtt.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-mqtt/src/main/resources/greengrass/features/mqtt.feature): Validates that the device can subscribe and publish to AWS IoT Core MQTT topics.
+
+
+
+- Below test features are only supported for Linux-based Greengrass core devices.
+  - [streammanager.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-streammanager/src/main/resources/greengrass/features/streammanager.feature): Validates that the device can download, install, and run the AWS IoT Greengrass stream manager.
+  - [docker.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-docker/src/main/resources/greengrass/features/docker.feature): Validates that the device can download a Docker container image from Amazon ECR.
+  - [ml.feature](../aws-greengrass-testing-features/aws-greengrass-testing-features-ml/src/main/resources/greengrass/features/ml.feature): Validates that the device can perform ML inference using the Deep Learning Runtime and TensorFlow Lite ML frameworks.
```

Comment:
Did you confirm this with anyone? I was not aware that they are not supported for Windows.

Reply:
Yes, they are not supported for Windows